armi/reactor/components/component.py

```diff
@@ -180,6 +180,10 @@ class Component(composites.Composite, metaclass=ComponentType):
         Temperature in C to which dimensions were thermally-expanded upon input.
     material : str or material.Material
         The material object that makes up this component and give it its thermo-mechanical properties.
+
+    .. impl:: All components in ARMI can be thermally-expanded using custom curves.
```

Comment:
not to be too nit picky with these sample tags, but here again I feel like it should say "here's where the thermal expansion is applied" (not actually here though)